src/sentry/utils/data_filters.py

```diff
@@ -0,0 +1,48 @@
+"""
+sentry.utils.data_filters.py
+~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import fnmatch
+
+
+class FilterTypes(object):
+    ERROR_MESSAGES = 'error_messages'
+    RELEASES = 'releases'
+    BLACKLISTED_IPS = 'blacklisted_ips'
+
+
+def is_valid_release(release, project):
```

Comment:
I'm happy to make that change because these functions are new, but since this is the way `is_valid_ip(ip_address, project)` is structured, would it be better to keep these consistent with that, change that as well (seems bad), or just change these two new ones?